packages/ui/src/lib/layouts/Page.svelte

```diff
@@ -34,16 +63,6 @@ function updateHeight(): void {
   heightOfDetail = detailSlot ? detailSlot.getBoundingClientRect().height : 0;
 }
 
-const dispatchClose = createEventDispatcher<{ close: undefined }>();
-export let onclose: () => void = () => {
-  dispatchClose('close');
-};
-
-const dispatchBreadCrumb = createEventDispatcher<{ breadcrumbClick: undefined }>();
-export let onbreadcrumbClick: () => void = () => {
```

Comment:
hello, I don't see the dispatcher still being called in case we don't provide a custom callback ?

Reply:
my bad, I missed this logic. I changed

Reply:
thanks